.github/pull_request_template.md

```diff
@@ -5,26 +5,28 @@
 ## Description
 <!-- Workshop description that you'll use in advertisements -->
 
+
+# Tasks
+More detailed instructions for specific steps here: <https://cct-datascience.github.io/group-procedures/workshops.html>
+
 ## Pre-workshop checklist:
 
-- [ ] Create a Zoom meeting with registration
+- [ ] Create a Zoom meeting with registration ([instructions & settings](https://cct-datascience.github.io/group-procedures/workshops.html#creating-a-template-for-workshop-zoom-meetings))
 - [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
 - [ ] Advertise
-  - [ ] Create event on [group website workshops page](https://datascience.cals.arizona.edu/workshops)
-  - [ ] Add to [Data Science Institute calendar](https://datascience.arizona.edu/calendar)
-  - [ ] Send out announcement to mailing list
-  - [ ] Share on resBazAZ and UA Data Science Slack
+  - [ ] Add an event to the [group website workshops page](https://datascience.cals.arizona.edu/workshops)
+  - [ ] Add to Data Science Institute calendar <https://datascience.arizona.edu/calendar>
+  - [ ] Include announcement in the next mailing list email
+  - [ ] resBazAZ and UA Data Science Slack
 - [ ] Provide homework / background reading as appropriate (installation, references, etc.)
 - [ ] Send out reminders
 
-Full checklist here: https://cct-datascience.github.io/group-procedures/workshops.html
-
 ## Materials
 
 Create a folder in the format YYYYMM-short-title and include:
```

Comment:
```suggestion
Create a folder on GitHub in the format YYYYMM-short-title and include:
```
It might seem obvious to specify GitHub, but it also rules out other options